### ISSUE TITLE
BLURRY - Only VSCode window is blurry?

### ISSUE THREAD

**Issue by classicrocker883** [microsoft/vscode#178540]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:  1.76.2
- OS Version:  Windows_NT x64 10.0.22621 (Windows 11 Pro 22H2 22621.1413) 

Steps to Reproduce:

1. Open VSCode
2. Use it as you would normally

The blur doesn't look like this all the time, when I move the mouse over any area ONLY on VSCode window the area clears up and shows as normal, then areas would go blurry again.

The screenshot shows only the VSCode window as blurry, however when viewing live through the monitor everything appears mostly normal, it will blur random areas of the vscode window and unblur when moving the mouse over.

This only happened in the latest or last update. It may have happened even sooner than that, but it wasn't always this way.
![vscodeblur](https://user-images.githubusercontent.com/18502096/228396126-206df2ad-05ef-4491-aec0-7d978a7cb0c9.png)

**Comment by rzhao271** [comment]
Hi, you can find previous versions at https://code.visualstudio.com/updates/v1_76.
I'm wondering what's the most recent previous version that worked for you?

**Comment by classicrocker883** [comment]
update ; it appears to be video driver related. I tried older vs code versions and the blur still happens. I tried it with the on board graphics through the motherboard back panel and there was no issues no blur.

so it's related to my graphics card/driver. I may try an older version of the driver. but what is strange is this **BLUR** only happens when using VS Code. it could also be an extensionin vs code causing the problem

**Comment by rzhao271** [comment]
Is the issue mitigated when you run cmd or Windows Powershell and type `code --disable-gpu` to launch VS Code?

**Comment by classicrocker883** [comment]
Yes that worked! I also went into Nvidia control panel and disabled/switched some `3D Settings` . when I updated the drivers I must have changed some global variables, VS Code is the only noticeable difference. I believe it may have to do with Antialiasing or Vertical Sync (V-sync).

thanks for your help @rzhao271 

the blur was really bugging me, literally.
Now as to fixing this that is out of my understanding. I did try a different graphics card and the same issue was there, so this is obviously something to do with the nvidia drivers or like i said antialiasing/v-sync. id do some more tests by seeing which parameter(s) caused it. for now problem solved. but if i can help whoever may figure out the exact code that caused this let me know.


![vscodeblur-nvidia](https://user-images.githubusercontent.com/18502096/228994975-7f31928d-9466-4150-b96a-599749c41708.png)
